src/Illuminate/Database/Eloquent/Concerns/HasAttributes.php

```diff
@@ -214,6 +216,39 @@ public function attributesToArray()
         return $attributes;
     }
 
+    /**
+     * Set attributes has append to the appends array.
+     *
+     * @return void
+     */
+    public function sutAppendMutatedAttributes()
```

Comment:
Should this not be `setAppendMutatedAttributes`?

```suggestion
    public function setAppendMutatedAttributes()
```